_posts/2019-02-27-partner-attributes.markdown

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Thoughts. What if we wanted to make this a little more flexible.
`/v1/partner_mapping/partner_company/:id` would be a request to "get the Gusto company mapped to this partner company."

Or

`/v1/partner_mapping/gusto_company/:id` would be a request to get the partner company_id mapped to the gusto company_id. 

The response payload, then, might reinforce the mapping nature of the endpoint. Something like:
```
{
  gusto: {
    company_id: 
    company_name:
  },
  partner: {
    company_id:  
  },
  mapped_at: 
}
```